GenderPayGap.WebUI/Controllers/Admin/AdminDownloadsController.cs

```diff
@@ -109,28 +109,33 @@ public FileContentResult DownloadOrphanOrganisations()
         [HttpGet("downloads-new/organisation-addresses")]
         public FileContentResult DownloadOrganisationAddresses()
         {
-            DateTime pinExpiresDate = Global.PinExpiresDate;
-
             List<Organisation> organisationAddresses = dataRepository.GetAll<Organisation>()
                 .Where(org => org.Status == OrganisationStatuses.Active)
-                .Where(org => org.LatestAddress != null)
-                .Include(org => org.LatestAddress)
+                .Include(org => org.OrganisationAddresses)
+                // This .ToList() materialises the collection (i.e. runs the SQL query)
+                .ToList()
+                // We only want organisations with valid addresses
+                // The following filter only works in code (cannot be converted to SQL) so must be done after the first .ToList()
```

Comment:
That's good to know, thanks for the handy note. Probably worth bringing this to @aidayounesi 's attention too